javascript/code-challenges/challenges-03.test.js

```diff
@@ -0,0 +1,308 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function called addTwo that takes in an array and adds two to every value using a for loop. Place the new value in a new array. Return the new array.
+------------------------------------------------------------------------------------------------ */
+
+const addTwo = (arr) => {
+  // Solution code here...
+  let plusTwo=[];
+  for(let i=0; i<arr.length; i++){
+    plusTwo.push((num) => num + 2);
+  } console.log(plusTwo);
+  return plusTwo;
+};
+
```

Comment:
You were very close on this one! 

```suggestion
const addTwo = (arr) => {
  let plusTwo=[];
  
  for(let i=0; i<arr.length; i++){
    plusTwo.push(arr[i]+ 2);
  } 
  return plusTwo;
};
```